turbo.json

```diff
@@ -1,22 +1,14 @@
 {
   "$schema": "https://turborepo.org/schema.json",
-  "baseBranch": "origin/master",
```

Comment:
Dangling property that is not supported by turbo. Seems to be a changeset config.json property.